src/test/java/fr/paris/lutece/plugins/workflow/modules/appointmentants/business/TaskAntsAppointmentConfigTest.java

```diff
@@ -1,18 +1,23 @@
-    package fr.paris.lutece.plugins.workflow.modules.appointmentants.business;
+package fr.paris.lutece.plugins.workflow.modules.appointmentants.business;
 
-    import junit.framework.TestCase;
 
-    public class TaskAntsAppointmentConfigTest extends TestCase {
-        public void testTaskAntsAppointmentConfig() {
-            TaskAntsAppointmentConfig config = new TaskAntsAppointmentConfig();
+import fr.paris.lutece.test.LuteceTestCase;
 
-            int formId = 1;
-            int fieldEntryId = 2;
+public class TaskAntsAppointmentConfigTest extends LuteceTestCase {
 
-            config.setIdForm(formId);
-            config.setIdFieldEntry(fieldEntryId);
+    public void setUp() {
```

Comment:
Why did you override this method? Not doing so prevents multiple initializations of Lutece-Core services when running tests.